src/foam/nanos/theme/Theme.js

```diff
@@ -519,7 +519,18 @@ foam.CLASS({
       name: 'spid',
       class: 'Reference',
       of: 'foam.nanos.auth.ServiceProvider',
-      value: foam.nanos.auth.ServiceProviderAware.GLOBAL_SPID
+      documentation: `
+        Only set spid on the main theme so that it can be looked up via spid
+        when could not find theme via domain.
+
+        By default, theme.spid is empty to denote an auxiliary theme which is
```

Comment:
Why is it called "auxiliary"?

Reply:
auxiliary because it's only for augmenting the other theme.

Reply:
Fixed https://github.com/foam-framework/foam2/pull/4125/commits/9a9d9c5c4f163a1545165f77e926b5ff08030465 - use "*" as default spid for theme.